keras_hub/src/models/backbone.py

```diff
@@ -188,11 +194,13 @@ def save_to_preset(self, preset_dir):
 
     def get_lora_target_names(self):
         """Returns list of layer names which are to be LoRA-fied.
-
-        Subclasses can override this method if the names of layers to be
-        LoRa-fied are different.
         """
-        return ["query_dense", "value_dense", "query", "value"]
+        return self._lora_target_names
+    
+    def set_lora_target_names(self, target_names):
+        """Set the list of layer names which are to be LoRA-fied.
+        """
+        self._lora_target_names = target_names
 
     def enable_lora(self, rank, target_names=None):
```

Comment:
I'd rename this to `target_layer_names`, it's a little more descriptive.

We should also document the args to `enable_lora` here, this is user facing. In the description for `target_layer_names` describe that `None` will populate this with target names with the models defaults, as returned by `backbone.default_lora_layer_names()`.